modules/admin-ui/src/main/webapp/scripts/shared/services/statsService.js

```diff
@@ -117,13 +111,6 @@ angular.module('adminNg.services')
         }
       };
 
-      // Reload the stats if the language is changed
```

Comment:
I'm confused - why don't we want to do this anymore?

Reply:
This file seems to be a copy-paste from the table service. The table service requires this because some localization work is done in the model ("resource"). 

For the stats service this was never required as the stats service has not localization logic in its model as the model is simply numbers.

-> We never needed this.

Reply:
Ah, I see!